### DISCUSSION TITLE
Request for RTL Support for Immersive Translate Extension

### DISCUSSION THREAD

**Discussion by fontgah** [immersive-translate/immersive-translate#371] [Ideas]
Dear Immersive Translate Development Team,

I recently discovered the Immersive Translate Chrome extension and found it to be an excellent tool that fulfills my translation needs. However, I noticed that the extension does not support RTL (Right-to-Left) text alignment for languages such as Persian, Arabic, and Afghan. This results in text alignment issues and makes it difficult for users to read translated text.

Therefore, I would like to request the addition of RTL support for the Immersive Translate extension. This feature would greatly benefit users who require RTL language support and enhance the usability and accessibility of the extension.

Here are some relevant points to consider:

RTL language support is essential for accurate and clear text alignment in translations for languages such as Persian, Arabic, and Afghan. Currently, there are no reliable alternatives that provide this functionality for the Immersive Translate extension.

Sure, here's a suggested message:

I recently reached out to the developer of ChatGPTBox and requested the addition of RTL support to their extension. They were able to successfully implement this feature and have it available in their latest release 

https://github.com/josStorer/chatGPTBox/issues/6

I wanted to bring this to your attention as well, since I believe adding RTL support to Immersive Translate would greatly benefit users who require this feature. ChatGPTBox integrated Google's text direction API to add RTL support, which could be a possible solution for Immersive Translate as well.

I hope you consider my request and add RTL support to the Immersive Translate extension.

Thank you for your attention to this matter.
Sincerely,

**Comment by theowenyoung** [comment]
Hi, Thanks for the details~

I'm happy to support RTL, but I think I need someone who knows Persian or Arabic language to contribute this feature. do you want to make some pull request for it?
